src/internal/components/file-input/interfaces.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BaseComponentProps } from '../../base-component';
+import { FormFieldCommonValidationControlProps } from '../../context/form-field-context';
+import { NonCancelableEventHandler } from '../../events';
+
+export interface FileInputProps extends BaseComponentProps, FormFieldCommonValidationControlProps {
+  /**
+   * Variant of the file input. Defaults to "button".
+   */
+  variant?: 'button' | 'icon';
+
+  /**
+   * Adds `aria-label` to the file input element. Use this to provide an accessible name for file inputs
+   * that don't have visible text, and to distinguish between multiple file inputs with identical visible text.
+   */
+  ariaLabel?: string;
+
+  /**
+   * Text displayed in the file input component. Used as the aria label for the button variant.
+   * @displayname text
+   */
+  children?: string;
```

Comment:
Minor: This is used as aria label if ariaLabel is not defined, not sure if we need to clarify that

Reply:
Updated to be more clear.